docs/clients/dovecot.txt

```diff
@@ -45,6 +45,25 @@ Configuring Dovecot to work with Virtual Exim
    Dovecot expects the select to return a set of columns with particular
    names, hence usage of the 'AS' keyword above.
 
-3. There's no step 3! Just restart Dovecot, and it should work.
+3. Exim can’t handle SHA512-crypt schemes for passwords, therefore you can use
+   the dovecot authenticator to login:
+   edit conf.d/10-master.conf
```

Comment:
This is a dovecot-configuration file. We should probably link the original documentation too
http://wiki2.dovecot.org/HowTo/EximAndDovecotSASL